corekit/src/main/java/com/midtrans/sdk/corekit/core/MerchantRestAPI.java

```diff
@@ -40,7 +40,7 @@ void paymentUsingCard(
      * @param requestModel SnapToken RequestModel
      */
     @Headers({"Content-Type: application/json", "Accept: application/json"})
-    @POST("/charge")
+    @POST("/installment/charge")
```

Comment:
Can you change back this ?
Instead of making changes on this, you can change the merchant base URL